lib/cli.js

```diff
@@ -280,7 +282,14 @@ exports.run = async () => { // eslint-disable-line complexity
 		});
 		watcher.observeStdin(process.stdin);
 	} else {
-		const runStatus = await api.run(files);
+		const {filenames, ranges, ignored} = rangeParser.parseFileSelections(files);
```

Comment:
What do you think about executing this when `cli.input` is processed? Then in the `conf.watch` branch we can exit with an error if lines are selected.